src/editor/contentState/updateCtrl.js

```diff
@@ -407,16 +408,17 @@ const updateCtrl = ContentState => {
         const preInputChar = text.charAt(+offset - 2)
         const postInputChar = text.charAt(+offset)
         /* eslint-disable no-useless-escape */
+        // Not Unicode aware, since things like \p{Alphabetic} or \p{L} are not supported yet
         if (
-          (autoPairQuote && /["]{1}/.test(inputChar)) ||
+          (autoPairQuote && /['"]{1}/.test(inputChar) && !(/[a-zA-Z\d]{1}/.test(preInputChar))) ||
```

Comment:
First of all, thank you very much for your PR. As discussed in the issue, we may only need to do this with single quotes. Double quotes need auto pair under any circumstances？

Reply:
After thinking about this for a while, I suppose we can let double-quotes auto-pair under any circumstances.